src/main/java/build/buildfarm/admin/Admin.java

```diff
@@ -29,4 +29,6 @@ void scaleCluster(
       Integer maxHosts,
       Integer targetHosts,
       Integer targetReservedHostsPercent);
+
+  void disableHostProtection(String scaleGroupName, String hostId);
```

Comment:
should there be an API to enable host protection?  How does enabling happen?

Reply:
I remember Yuriy mentioned that all hosts/instances will have it enabled as default when they are started.